packages/opentelemetry-exporter-prometheus/src/PrometheusSerializer.ts

```diff
@@ -177,6 +178,14 @@ export class PrometheusSerializer {
 
   serializeRecord(name: string, record: MetricRecord): string {
     let results = '';
+
+    if (
+      record.descriptor.metricKind === MetricKind.COUNTER &&
+      !name.endsWith('_total')
+    ) {
+      diag.debug(`Counter ${name} is missing the mandatory _total suffix`);
```

Comment:
This is likely extremely chatty. If a counter has a wrong suffix it will log every time it is exported. Would like to see a way to log only once per bad name.

Reply:
if we keep the information about used suffixes the memory can grow endlessly :/, maybe limit the number of this particular information in total ? Although I'm hesitant to say that we should also considering leaving the decision for the end user. By default nothing will be shown, but it is up to the user to decide what to do with this, maybe someone want's to see all errors. Btw shouldn't this be a `warning` instead of `debug` ?

Reply:
if it is mandatory, we could potentially append the string with `_total` if not present? I think this is done in opentelemetry-collector does for the prometheus remote write exporter: https://github.com/open-telemetry/opentelemetry-collector/blob/03c7bf98f7edf6e578aac303e4d8b767f117541e/exporter/prometheusremotewriteexporter/helper.go#L205-L207

Reply:
That's a good idea @naseemkullah. If it's already collector's behaviour maybe we should match it and maybe log a warning once?

Reply:
@naseemkullah I have implemented your suggestion and updated the tests accordingly